### DISCUSSION TITLE
[DOC] Integrator's doc is not cristal clear on the definition of `PGSCHEMA` and `PGSCHEMA_STATIC` variables in the `env.project` file.

### DISCUSSION THREAD

**Discussion by nicolas-sitylb** [camptocamp/c2cgeoportal#10849] [Q&A] [answered]
Hello,

It's not 100% clear to me what value the two variables `PGSCHEMA` and `PGSCHEMA_STATIC` should take in the `env.project` file when [upgrading the GMF application](https://camptocamp.github.io/c2cgeoportal/master/integrator/upgrade_application.html#prepare-the-database):

https://github.com/camptocamp/c2cgeoportal/blob/46f185fdf72aade463c8d4add0e68c78d6bbfa52/doc/integrator/upgrade_application.rst?plain=1#L34

Could the Integrator's documentation be a little more explicit here?

Thank you.

**Comment by jwkaltz** [comment, marked as answer]
Hi,
this part of the documentation has now been improved, as per #10850.

**Comment by nicolas-sitylb** [comment]
Hi @jwkaltz,
the [c2cgeoportal/doc/integrator/upgrade_application.rst](https://github.com/camptocamp/c2cgeoportal/blob/46f185fdf72aade463c8d4add0e68c78d6bbfa52/doc/integrator/upgrade_application.rst?plain=1#L34) file was not modified by https://github.com/camptocamp/c2cgeoportal/pull/10850. 
Hence, the original question about the GMF upgrade remains open.

Btw thanks for updating the other files.

**Comment by sbrunner** [comment]
The upgrade invites you to create new schemas, then you should put the new schemas in the `env.project` file :-)
